src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$primary: #5E60CE;
```

Comment:
Why you choose to not use styled components ?

Reply:
Since it is start of the project, I can switch to styled components and typescript.